web3/utils/events.py

```diff
@@ -238,3 +253,168 @@ def remove_trailing_from_seq(seq, remove_value=None):
 normalize_topic_list = compose(
     remove_trailing_from_seq(remove_value=None),
     pop_singlets,)
+
+
+def is_indexed(arg):
+    if isinstance(arg, TopicArgumentFilter) is True:
+        return True
+    return False
+
+
+not_indexed = complement(is_indexed)
+
+
+class EventFilterBuilder:
+    formatter = None
+    _fromBlock = None
+    _toBlock = None
+    _address = None
+
+    def __init__(self, event_abi, formatter=None):
+        self.event_abi = event_abi
+        self.formatter = formatter
+        self.event_topic = self._initial_event_topic()
+        self.args = AttributeDict(_build_argument_filters_from_event_abi(event_abi))
+        self._ordered_arg_names = tuple(arg['name'] for arg in event_abi['inputs'])
+
+    @property
+    def fromBlock(self):
+        return self._fromBlock
+
+    @fromBlock.setter
+    def fromBlock(self, value):
+        self._fromBlock = value
+
+    @property
+    def toBlock(self):
+        return self._toBlock
+
+    @toBlock.setter
+    def toBlock(self, value):
+        self._toBlock = value
+
+    @property
+    def address(self):
+        return self._address
+
+    @address.setter
+    def address(self, value):
+        self._address = value
```

Comment:
Since the getter and setter are both just the defaults, why not just use a normal class attribute and drop the `property` stuff.  Same with the rest of these `property` functions.  If they don't do any special functionality and have both a `getter` and `setter` then a normal property is probably easier to reason about.